website/templates/project/modal_add_contributor.mako

```diff
@@ -148,7 +148,7 @@
                                 </thead>
                                 <tbody data-bind="sortable: {data: selection, as: 'contributor', afterRender: makeAfterRender(), options: {containment: 'parent'}}">
                                     <tr>
-                                        <td style="padding-right: 10px;">
+                                        <td style="padding-right: 10px; width: 50px;">
```

Comment:
Only `padding-right:10px` could be replaced by class. I will add that class instead.